cartography/intel/aws/iam.py

```diff
@@ -74,13 +74,18 @@ def get_group_managed_policy_data(boto3_session, group_list):
 
 @timeit
 def get_user_policy_data(boto3_session, user_list):
+    client = boto3_session.client('iam')
     resource_client = boto3_session.resource('iam')
     policies = {}
     for user in user_list:
         name = user["UserName"]
         arn = user["Arn"]
         resource_user = resource_client.User(name)
-        policies[arn] = {p.name: p.policy_document["Statement"] for p in resource_user.policies.all()}
+        try:
+            policies[arn] = {p.name: p.policy_document["Statement"] for p in resource_user.policies.all()}
+        except client.exceptions.NoSuchEntityException:
```

Comment:
I think we can catch `botocore.errorfactory.NoSuchEntityException` instead. This way we don't need to instantiate a boto3 client that isn't actually used.

Reply:
Yup this makes sense to me. Just seen that you've caught aws exceptions in different ways in this project - wasn't too sure which way you wanted to go with here :) I'll change to botocore :)

Reply:
@mdrdannyr Oh actually, use [this technique](https://stackoverflow.com/a/44526994) to catch the exception from a resource. My bad; it seems you can't actually import `botocore.errorfactory.*`.

Reply:
Yup already committed with that method. Thought it was better as well :)